### PR TITLE
Ilan/update staging json

### PR DESCRIPTION
**Pull request by ilanDoron**
sanity rates new contract + operator.

fee burner - change admin to wrapper.
update max per block and max total for 3 coins.

### CODE REVIEW

File: contracts/DigixConnector.sol [hunk 1]
```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.0;
```
Comment (yaronvel) [inline contracts/DigixConnector.sol]: is this file relevant to the PR? please remove it

**Comment by ilanDoron** [comment]
done.
removed files.
